packages/manager/modules/pci/src/projects/project/instances/instances.controller.js

```diff
@@ -34,6 +40,77 @@ export default class CloudProjectComputeInfrastructureListCtrl {
     this.checkHelpDisplay();
 
     this.criteria = getCriteria('id', this.instanceId);
+
+    this.fetchInstances();
+  }
+
+  fetchInstances() {
+    this.loading = true;
+    return this.$q
+      .all({
+        instances: this.PciProjectsProjectInstanceService.getAll(
+          this.projectId,
+          this.customerRegions,
+        ),
+        floatingIps: this.getFloatingIps(),
+      })
+      .then(({ instances, floatingIps }) => {
+        const updatedInstances = instances.map((instance) => ({
+          ...instance,
+          floatingIp: floatingIps.find(
```

Comment:
Do we assume later that `floatingIp` could be `undefined`?

Reply:
It was a copy from previous call, so I think yes